blobstore/cmd/flynn-blobstore-cleanup/cleanup.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"sync"
+
+	"github.com/flynn/flynn/blobstore/data"
+	"github.com/flynn/flynn/pkg/postgres"
+)
+
+func main() {
+	concurrency := flag.Int("concurrency", 4, "number of parallel file deletions to run at a time")
+	flag.Parse()
+
+	db := postgres.Wait(nil, nil)
+	repo, err := data.NewFileRepoFromEnv(db)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Not necessary, the TCP connection will be closed by the OS when the program exits.